.mocharc.js

```diff
@@ -0,0 +1,13 @@
+// we can't use braces glob as it's expanded by the shell and not mocha
+function testGlob(suffix) {
+  const base = 'node-tests/';
+  const dirs = ['blueprints', 'acceptance', 'unit'];
+
+  return dirs.map(dir => `${base}${dir}${suffix}`);
+}
+
+module.exports = {
+  timeout: 5000,
+  reporter: 'spec',
+  spec: testGlob('/**/*-test.js'),
```

Comment:
I think we could make this much simpler by copy pasting the three globs

Reply:
e.g. 
```
spec: [
  'node-tests/blueprints/**/*-test.js',
  'node-tests/acceptance/**/*-test.js',
  'node-tests/unit/**/*-test.js'
]
```

Reply:
Yes that's what I was thinking about.
I've updated the PR with it because it's much easier to understand now.